open-company/open-company-storage
# Fix viewer contributor access

Bug: if you setup a viewer with author access to a private section you will see that:
- no edit/delete entry links are sent (at least on AP api)
- publish is not allowed (create post and save are though)
- share is not allowed
- drafts board is not present

To test:
- with user A that is an admin of the org
- add a viewer (user B) to your org
- add 2 private sections (section 1 and section 2)
- make the viewer an author on Section 1 only
- with user A
- make sure you get edit/delete links in the menu in All posts for activities on all boards
- make sure you get edit/delete links in the menu in section 1 and section 2
- with user B
- go to All posts
- make sure you get Share only for team sections
- make sure you get edit/delete links for Section 1
- make sure you get only share for Section 2 posts
- now go to Section 1
- make sure you get edit/delete
- now go to Section 2
- make sure you get only share
- make sure user B sees drafts board
- make sure you can create posts only in Section 1
- make sure you can move posts only to Section 1
- make sure you can remind a use of a post in Section 1
